client/src/main/scala/skuber/api/package.scala

```diff
@@ -795,6 +814,9 @@ package object client {
 
     def durationFomConfig(configKey: String): Option[Duration] = Some(Duration.fromNanos(appConfig.getDuration(configKey).toNanos))
     val watchIdleTimeout: Duration = getSkuberConfig("watch.idle-timeout", durationFomConfig, Duration.Inf)
+    // Timeout needs to be less than idle connection timeout.
+    // Max timeout set to 60 seconds.
```

Comment:
Is there a specific reason for max of 60 seconds? Can this be configurable?

Reply:
Was looking to use "watch.idle-timeout" as a basis for determining what the "maxWatchRequestTimeout" is as it has to be a smaller value. However, as it is currently possible to set "watch.idle-timeout" to a value Duration.Inf we need a sensible default value for maxWatchRequestTimeout in those cases. Could it be configurable yes... though i would think it would be confusing. 

Given that we are looking to have a separate connection pool it is probably a better option to have something like the following in the [config](https://github.com/doriordan/skuber/blob/master/client/src/main/resources/reference.conf) 
```
skuber {

  watch-continuously {
    request-timeout = 30s
  }
}
```
and avoid overloading the current config.